Dockerfile

```diff
@@ -26,8 +26,9 @@ RUN sed -i 's#"./src/cli"#"./lib/cli"#g' package.json
 
 EXPOSE 3030
 
-ENTRYPOINT [ "./bin/run" ]
-CMD [ "start" ]
+ENV RIFM_PRECACHE_TARGETS rns rates
```

Comment:
Hmmm I am pretty sure that `ENV` is only for build-time and not run-time. So I would suggest removing this. Btw. there is also an option "all" which will precache all the services, which I would make a default if the `RIFM_PRECACHE_TARGETS` is not set.

Reply:
From https://docs.docker.com/engine/reference/builder/#env

> The environment variables set using `ENV` will persist when a container is run from the resulting image. You can view the values using `docker inspect`, and change them using `docker run --env <key>=<value>`.

Didn't know about `all`. Will use it instead.